### PR TITLE
fix: shopify app throw an error if collection or product is missing and list not render

### PR DESCRIPTION
**Pull request by vishalmishra090**
## Purpose

Shopify app throws an error if a collection or product is missing. Due to this List is not rendered and the user is not able to remove missing products

### CODE REVIEW

**Comment by rusticpenguin** [comment]
Hi @vishalmishra090! I appreciate you taking the time to make this PR! I am having trouble recreating the issue you're addressing in your code. 

Here are the steps I took:
1. Created a list of products in Contentful using the Shopify app
2. Deleted a product in that list in the Shopify Admin
3. Verified I was able to still see the list of products and remove the missing product

I have attached a screenshot of the working list:
image

If this is still an issue, could you give me steps to recreate the error so I can verify your PR!

Thanks!

File: apps/shopify/src/skuResolvers.js [hunk 1]
```diff
@@ -133,7 +133,7 @@ export const fetchProductPreviews = async (skus, config) => {
   }
 
   const response = (await Promise.all(requests)).flat();
-  const products = response.map((res) => convertProductToBase64(res));
+  const products = response.map((res) => res && convertProductToBase64(res));
```
Comment (rusticpenguin) [inline apps/shopify/src/skuResolvers.js:136]: Couldn't trigger an error with this line of code. Mine reverting this back to how it was?
```suggestion
  const products = response.map((res) => convertProductToBase64(res));
```

**Review by rusticpenguin — CHANGES_REQUESTED** [review]
Verified the Collections were erroring out. I couldn't get a product list to error out though. So feel free to revert that line and we'll be good to go on merging!

**Comment by matthew-gordon** [comment]
Closing this as `shopify` has moved to https://github.com/contentful/marketplace-partner-apps, I will come back and add the new PR reference when available, thank you!

File: apps/shopify/src/skuResolvers.js (hunk 1 above)
Comment (vishalmishra090) [reply apps/shopify/src/skuResolvers.js:136]: For Products, In-app Configuration, we have two options - Product and Product Variant, With Variant option It is working fine but when we select Product option then it throws an error

Comment (rusticpenguin) [reply apps/shopify/src/skuResolvers.js:136]: Sounds good! I see what you mean now! Mind making the PR to this repo: https://github.com/contentful/marketplace-partner-apps

We've moved the shopify app's code over here! I am having trouble applying your commit over there and want you to receive credit

Comment (vishalmishra090) [reply apps/shopify/src/skuResolvers.js:136]: I have opened a [PR](https://github.com/contentful/marketplace-partner-apps/pull/101) in the [new repo](https://github.com/contentful/marketplace-partner-apps). you can review it

**Pull request closed without merging**